web/src/app/modules/views/side/modules/errors-warnings/components/warning.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { IContainer } from '../../../../../../model/IContainer';
+import { ValidationResult } from '../../../../../../validation/validation-result';
+import { AdditionalInformationService } from '../../links-actions/services/additional-information.service';
+import { SelectedElementService } from '../../selected-element/services/selected-element.service';
+
+@Component({
+    selector: '[warning]',
+    templateUrl: 'warning.component.html',
+    styleUrls: ['warning.component.css']
+})
+
+export class Warning implements OnInit {
+    private valResults: ValidationResult[] = [];
```

Comment:
This is also used in the UI -> Please make public.

```suggestion
    public valResults: ValidationResult[] = [];
```